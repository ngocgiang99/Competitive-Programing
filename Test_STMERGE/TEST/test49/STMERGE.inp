4
2 5
857330614 110567430 52971368 540478548 962576741 
917514600 229600179 608916152 344559833 771695998 
3 4
920029544 366842414 956285173 722299873 
751868582 455741853 231947543 922901745 
593981544 993882159 569101503 518271215 
10 8
205152803 346583200 593225742 621500818 554923449 336846089 141125852 664511598 
598100532 897820527 955740674 175661184 155697290 680573272 841378453 993759088 
358873584 606514823 969803755 430294305 938335194 310345475 603253144 103212985 
770804735 666632464 494163825 53150620 947967571 607829811 859363212 208052592 
553416458 317944369 237975101 172860442 897754779 253336848 349570586 940657854 
880401849 533453587 834211319 944294082 857105754 231523777 14639850 846299595 
828735155 965583582 103645255 726257597 4905785 568714351 312633251 286800062 
385348257 316714169 664449747 17343794 469603039 319023169 973835757 231092159 
293036074 712492492 214356294 182654794 605257462 621249120 806066034 822557944 
629679607 33127232 430445894 218089005 834016291 398537715 377455575 250185870 
3 7
869262459 344577538 270598928 581796899 316626119 373202092 636109051 
522088938 298539663 52241137 684287589 13259288 783755240 716786040 
5475715 989459312 35317031 452436772 557340873 973589241 150439650 
