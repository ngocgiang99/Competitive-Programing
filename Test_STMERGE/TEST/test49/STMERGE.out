397531201
593981544
238280035
5475715
